### DISCUSSION TITLE
Data type support long integer

### DISCUSSION THREAD

**Discussion by hope-data-science** [duckdb/duckdb#6102] [Ideas]
When I transfer some data to duckdb, I find the the former integer64 can not be transfered correctly and the information has been lost. In the future will it be supported?

Thanks.

**Comment by Mytherin** [comment]
DuckDB supports int64. Could you be more specific on what you are trying to do and how it is failing?

**Reply by hope-data-science** [reply]
I use duckdb package and DBI package in R and use IO to save a data as duckdb file. When I save sqlite, it works well. But to duckdb, it fails and save any int64 as 8.68e-310.

**Comment by hannes** [comment]
I suspect this is R-related?

**Reply by hope-data-science** [reply]
This is R-related. When I use sqlite, it would be OK. When I use duckdb package to save to duckdb (also using DBI to connect), it all turns to the same number of 8.68e-310 .
